roles/servicetelemetry/tasks/_local_signing_authority.yml

```diff
@@ -8,6 +8,7 @@
         name: '{{ ansible_operator_meta.namespace }}-selfsigned'
         namespace: '{{ ansible_operator_meta.namespace }}'
       spec:
+        duration: 70080h
```

Comment:
I wonder if this should be put in as a value read from `defaults/main.yml` just so we have a single spot to edit this in the future?

Reply:
Certainly!